deploy-manage/manage-spaces.md

```diff
@@ -23,4 +26,106 @@ $$$spaces-control-feature-visibility$$$
 
 $$$spaces-control-user-access$$$
 
-$$$spaces-managing$$$
\ No newline at end of file
+$$$spaces-managing$$$
+
+**Spaces** let you organize your content and users according to your needs.
+
+- **\[Not available in serverless]** Each space has its own navigation.
+- Each space has its own saved objects. 
+- Users can only access the spaces that they have been granted access to. This access is based on user roles, and a given role can have different permissions per space.
+
+{{kib}} creates a default space for you. When you create more spaces, users are asked to choose a space when they log in, and can change their current space at any time from the top menu.
+
+:::{image} ../../../images/kibana-change-space.png
+:alt: Change current space menu
+:class: screenshot
+:::
+
+To go to **Spaces**, find **Stack Management** in the navigation menu or use the [global search bar](../../../get-started/the-stack.md#kibana-navigation-search).
+
+
+## Required permissions [_required_privileges_3]
+
+The `kibana_admin` role or equivalent is required to manage **Spaces**.
+
+::::{note}
+In a serverless project, you must have an admin role on that project to manage its **Spaces**.
```

Comment:
> In a serverless project, you must have an admin role on that project to manage its **Spaces**.

Do we need to have this in an admonition? This seems unnecessarily eye-catching to me.